ui/src/components/stepforms/widgets/Autocomplete.vue

```diff
@@ -100,6 +108,10 @@ export default class AutocompleteWidget extends FormWidget {
   updateValue(newValue?: string | object) {
     this.$emit('input', newValue);
   }
+
+  onOptionClick(e: Event, disabled?: boolean) {
+    if (disabled) e.stopPropagation();
+  }
```

Comment:
It's smart to have used the stopPropagation here :+1: The selection logic from vue-multiselect seems not duplicated :clap: